app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split = command.split(" ")
```

Comment:
The variable name `split` is not descriptive. Consider renaming it to something like `command_parts` to better convey its purpose.